### PR TITLE
Generate translatable test value

### PR DESCRIPTION
**Pull request by SebaMutuku**

### CODE REVIEW

**Comment by dubdabasoduba** [comment]
I this should also be added to the general app `develop | master` this is a general app update.

File: opensrp-anc/src/main/java/org/smartregister/anc/library/util/ANCFormUtils.java [hunk 1]
```diff
@@ -185,20 +185,6 @@ private static void processCheckBoxSpecialWidget(JSONObject widget, List<String>
...
-            JSONArray options = jsonObject.getJSONArray(JsonFormConstants.OPTIONS_FIELD_NAME);
-            JSONObject selectedOption = formUtils.getOptionFromOptionsUsingKey(options, value);
-            newValue.put(JsonFormConstants.VALUE, value);
-            newValue.put(JsonFormConstants.TEXT, selectedOption.optString(JsonFormConstants.TRANSLATION_TEXT, ""));
-            return newValue;
-        }
-        newValue.put(JsonFormConstants.VALUE, value);
-        newValue.put(JsonFormConstants.TEXT, jsonObject.optString(JsonFormConstants.TRANSLATION_TEXT, ""));
-        return newValue;
-    }
```
Comment (dubdabasoduba) [inline opensrp-anc/src/main/java/org/smartregister/anc/library/util/ANCFormUtils.java:201]: Any reason this was moved from one Utils file to another?

**Review by dubdabasoduba — CHANGES_REQUESTED** [review]

File: opensrp-anc/src/main/java/org/smartregister/anc/library/util/ANCFormUtils.java (hunk 1 above)
Comment (SebaMutuku) [reply opensrp-anc/src/main/java/org/smartregister/anc/library/util/ANCFormUtils.java:201]: None. Just saw it looked better under Utils section

Comment (SebaMutuku) [reply opensrp-anc/src/main/java/org/smartregister/anc/library/util/ANCFormUtils.java]: This was done

**Review by dubdabasoduba — CHANGES_REQUESTED** [review]
@SebaMutuku Please fix code standards here

**Pull request closed without merging**